modules/vpc/variables.tf

```diff
@@ -0,0 +1,31 @@
+# Input variable definitions
+
+variable "public_cidr_block" {
+  description = "Range of the cidr block for public subnets"
+  type        = list(string)
+}
+
+variable "private_cidr_block" {
+  description = "Range of the cidr block for private subnets"
+  type        = list(string)
+}
+
+variable "az" {
+  description = "Range of the available zones"
+  type        = list(string)
+}
+
+variable "vpc_cidr_block" {
+  description = "The cidr block of vpc"
+  type        = string
+}
+
+variable "public_number" {
```

Comment:
the "public_number" variable will not be needed after you fix my comments regarding count

Reply:
fixed